database/migrations/2025_04_11_130811_create_educator_resources_translation_fields.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('educator_resource_translations', function (Blueprint $table) {
+            createDefaultTranslationsTableFields($table, 'educator_resource');
+            $table->string('title', 200)->nullable();
+            $table->string('title_display')->nullable();
+            $table->text('listing_description')->nullable();
+            $table->text('short_description')->nullable();
+            $table->string('meta_title')->nullable();
+            $table->text('meta_description')->nullable();
```

Comment:
These database columns should be in only one table: either `educator_resource` OR `educator_resource_translations`. Keep this migration as is, but create a new migration that does the following:

* Copy the contents of each of these fields from the `educator_resource` to the `educator_resource_translations` table
* Drop these columns from the `educator_resource` table